### PR TITLE
Dependency update

### PR DESCRIPTION
**Pull request by maskarb**
## Description

* remove pytz
* reinstall tzdata in the Dockerfile 
* update dependencies

### CODE REVIEW

**Review by djnakabaale — APPROVED** [review]
lgtm 👍🏿

File: koku/masu/external/date_accessor.py [hunk 1]
```diff
@@ -6,14 +6,16 @@
...
+from zoneinfo import ZoneInfoNotFoundError
 
-import pytz
 from dateutil import parser
 from dateutil.relativedelta import relativedelta
 
 from masu.config import Config
 
 LOG = logging.getLogger(__name__)
+UTC = ZoneInfo("UTC")
```
[the remark this answers is not in the document]
Comment (djnakabaale) [reply koku/masu/external/date_accessor.py]: this is addressed in https://github.com/project-koku/koku/pull/4397

File: Pipfile.lock [hunk 2]
```diff
@@ -956,43 +956,72 @@
...
+                "sha256:df22bd073107fe20006f9197ba29579e915a9a89142ef65130939aa0136c78e7",
+                "sha256:dfbd7db8ed8ce8ab0cb1dee1a25f15a900c9e66ef12004ab593984a51ba36fae",
+                "sha256:e08ebbdfa008f5d957e69031ab625f7d22313fdff14f845d213ab06f585846e4",
+                "sha256:e390c820a31146c73b0cb60da65906a866d4fe43b4688cc3cdf5f33c423d09a1",
+                "sha256:e6e5e4f543bf798161321deb186723a4ef530e7f216ea4c153e525f63c78fc78",
+                "sha256:ec5c0e0f4be15d73d2ff8d00212c67fcd4f36bca4b43e3c940c545341805dd22",
+                "sha256:ec8ba7a8357a33af6c41a29a3015f796643e998a1a33d95cb773efc98ae669d2",
+                "sha256:f4ec497d5926842a45f235d1eda03ef2cc10e4ccb0738bbe828837817f0fbbfd"
+            ],
+            "version": "==4.0.0"
```
[the remark this answers is not in the document]
Comment (maskarb) [reply Pipfile.lock:1024]: So I've converted this PR to just be dependency updates. I think we can rely on smokes here. I'll also look at some release notes to see if there is anything we really need to be concerned with.

**Review by samdoran — APPROVED** [review]

File: Pipfile.lock (hunk 2 above)
[the remark this answers is not in the document]
Comment (maskarb) [reply Pipfile.lock:1024]: Just looking over the major dep updates, the only one I'm concerned about is the `tzlocal`, but that has been address by the `tzdata` reinstall.

**Pull request merged**